### PR TITLE
Add a test confirming comment for the contribution

### PR DESCRIPTION
**Pull request by mttrs**
This repository has travis. So, all test passed is required to build/merge.
https://travis-ci.org/yuki24/did_you_mean

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,8 +103,9 @@ OBject
 1. Fork it (http://github.com/yuki24/did_you_mean/fork)
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+4. Confirm all tests passed (`bundle exec rake`)
```
Comment (yuki24) [inline README.md]: `Make sure all tests pass` sounds more natural to me. Could you change this line?

**Comment by mttrs** [comment]
Updated. Could you confirm?

**Comment by yuki24** [comment]
Could you squash the commits into a single commit? Other than that, it looks good!

**Comment by mttrs** [comment]
@yuki24 , Now it's a single commit. Please check :)

**Comment by yuki24** [comment]
Thanks! :heart: :yellow_heart: :blue_heart:

**Pull request merged**